### ISSUE TITLE
Not able to match "eosio::bidname" actions.

### ISSUE THREAD

**Issue by rossgeesman** [EOSIO/demux-js#71]
I'm running the example code included in this repo except I have modified the effects array in attempt to match bidding on premium names. 
```
const effects = [
  {
    actionType: "eosio::bidname",
    run: console.log,
  },
]
```
I've been starting the NodeosActionReader so that it replays blocks where `eosio::bidname` actions definitely occur but I've been unable to successfully match them. Otherwise, my project works fine for other actions like `eosio.token::transfer`. Also logging the blockinfo passed through to my effects handler when matching other action types does show that `bidname` actions are included in the block actions array.

Perhaps I am misunderstanding something about how to use this project? I'd really appreciate any advice you can give.

**Comment by jeffreyssmith2nd** [comment]
Can you provide a block number to start at so I can try to reproduce the issue?

**Comment by rossgeesman** [comment]
@jeffreyssmith2nd The most recent block with the relevant action is #25977074 on mainnet. For reference the tx is here: https://eospark.com/MainNet/tx/bcfe61e678a597cc3becf72bd9f97aa1c05106b0ef3bfd996a56c62bbb9d81f0 Thanks!

**Comment by rossgeesman** [comment]
@jeffreyssmith2nd Thanks for the quick follow up.

On my original project (which I had copied from the examples folder in this repo) the versions were     `"demux": "^3.0.0"` and `"demux-eos": "^2.0.0"`. I was beginning to suspect that I had made some changes to the project when I was playing around and my code was somehow different from the examples/eos-transfers so I did the following:

1. `git clone git@github.com:EOSIO/demux-js.git`

2. From the project root, make sure I'm on branch `develop` then `cd examples/eos-transfers`.

3. Run `npm install`

4. Set starting block # on line 29 of index.js to 25977060, like so: 
```
const actionReader = new NodeosActionReader(
  "http://mainnet.eoscalgary.io", // Thanks EOS Calgary!
  25977060,
)
```
5. Replace the effects array with your example:
```
const effects = [
 {                                                                                                                                                                             
      actionType: "eosio::bidname",                                                                                                                                               
      run: (payload, blockInfo, context) => {                                                                                                                                     
      console.log("BIDNAME matched") 
      console.log(payload.data)                                                                                                                                                 
      console.log(blockInfo)                                                                                                                                                                                                                                                                                            
     },                                                                                                                                                                          
   },
]
```
6. From project root, run `npm run start`

It processes blocks but still never matches "eosio::bidname" so the effects aren't run and nothing gets logged.

**Comment by jeffreyssmith2nd** [comment]
I realized I had accidentally had it to the `updaters` array too which is why I saw it working. I'll try to dig into why it isn't working for the `effects`.

**Comment by jeffreyssmith2nd** [comment]
@rossgeesman I created a PR (#73 ) to fix this issue, we'll try to get that in soon.

**Comment by flux627** [comment]
Resolved by #73
